modules/planning/tasks/optimizers/path_time_heuristic/gridded_path_time_graph.cc

```diff
@@ -266,7 +266,8 @@ void GriddedPathTimeGraph::GetRowRange(const StGraphPoint& point,
 
   const auto max_s_size = dimension_s_ - 1;
 
-  const double max_acceleration = std::abs(vehicle_param_.max_acceleration());
+  const double max_acceleration = std::abs(
+                       gridded_path_time_graph_config_.max_acceleration());
```

Comment:
Use `./scripts/clang-format.sh` to fix style issues here.

Reply:
Great tools to use. Thank you!